docs/src/pages/docs/crocks/Maybe.md

```diff
@@ -88,6 +88,52 @@ bigNumber(3)
 
 </article>
 
+<article id="topic-construction">
+
+## Construction
+
+```haskell
+Maybe :: a -> Maybe a
+```
+
+Most of the time construction of a `Maybe` is accomplished through the use
+of helper functions like [`safe`](#safe) or by employing one of the instance
+constructors, [`Just`](#just) or [`Nothing`](#nothing). This is due to the
+nature of `Maybe` and most other Sum Types.
+
+As a matter of consistency and completion, a `Maybe` instance can also be
+constructed using its TypeRep like any other type. The `Maybe` constructor is a
+unary function that accepts any type `a` and returns a `Just` instance, wrapping
+the value passed to its argument.
+
+```javascript
+import Maybe from 'crocks/Maybe'
+import equals from 'crocks/pointfree/equals'
+
+Maybe('some string')
```

Comment:
I'd be careful with this, personally i think that given the inherent `Nothing`-ness of both `null` and `undefined` i feel that this constructor should actually return a `Nothing` if it receives either of those values. Only reason i havn't brought it up is at this point i am sure that feature has made it into more than a few code bases

Reply:
@dalefrancis88 There is a problem with that line of reasoning.

First off, Constructors should construct something, so for all the Monadic Sum Types (excluding `Async` for obvious reasons) they default to `of` behavior. While doing the `Nothing` construction on provided types makes sense for `Maybe`, it will differ from all other types as it does not implement the `of`. All other types, which have to specify a `left`-most value for their `left`-most instances, could not match.

To add this behavior to `Maybe` will lose the 🌽sistancy with the other Sum Types. Now we cannot modify the behavor of `of` for `Maybe` as it would no longer be an `Applicative` and as such could not be a `Monad`. The `Applicative` MUST lift an un-altered value (or function or whateves) blindly into the type. It is standard (although does not NEED to be) that for Sum Types `of` lifts into the right most instance.

Hope all of that makes sense!

Reply:
This makes sense, thanks for explaining